cwod_site/bioguide/management/commands/import_roles.py

```diff
@@ -57,5 +61,6 @@ def handle(self, *args, **options):
                 except IntegrityError:
                     continue
 
-            time.sleep(.55)
+            time.sleep(0.1)
```

Comment:
They didn't mention, according to ProPublica's website they have a limit of 5000 req/day. Other than that didn't see any limits. I can increase this back to their original value just wanted to speed up the process a bit while I was testing.